### PR TITLE
Audit npm dependencies

### LINKED ISSUE #98: Audit npm dependencies
Run `npm audit`

### PR DESCRIPTION
**Pull request by disrupted**
Closes #98

### CODE REVIEW

File: frontend/src/api/fetchers.tsx [hunk 1]
```diff
@@ -14,6 +14,22 @@ import {
 
 export type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
 
+const encodingFn = encodeURIComponent;
+
+const encodingTagFactory = (encodingFn: typeof encodeURIComponent) => (
```
Comment (VictorKuenstler) [inline frontend/src/api/fetchers.tsx:19]: Is it possible to disable this?

**Comment by VictorKuenstler** [comment]
Could you please check whether updating `react-scripts`  also solves some of the issues in the audit report?  I think a version that does not require upgrading React to 17 could be a possible option.

File: frontend/src/api/fetchers.tsx (hunk 1 above)
Comment (disrupted) [reply frontend/src/api/fetchers.tsx:19]: this seems to have been introduced by https://github.com/contiamo/restful-react/issues/289. All I could find the option `pathParametersEncodingMode` for a custom config file https://github.com/contiamo/restful-react#config-file-format

**Comment by disrupted** [comment]
> Could you please check whether updating `react-scripts` also solves some of the issues in the audit report? I think a version that does not require upgrading React to 17 could be a possible option.

updating react-scripts unfortunately doesn't solve any vulnerabilities for us

**Comment by VictorKuenstler** [comment]
> > Could you please check whether updating `react-scripts` also solves some of the issues in the audit report? I think a version that does not require upgrading React to 17 could be a possible option.
> 
> updating react-scripts unfortunately doesn't solve any vulnerabilities for us

Ok, thanks for checking!

**Review by VictorKuenstler — APPROVED** [review]
LGTM

**Comment by disrupted** [comment]
> I don't know how the title of this PR is related to the changes

This is for updating some of our underlying Node dependencies which had critical vulnerabilities, reported by `npm audit`.

**Pull request merged**